docs/integrations/sources/hubspot.md

```diff
@@ -9,21 +9,21 @@ You can use OAuth, API key, or Private App to authenticate your HubSpot account.
 | Stream                      | Required Scope                                                                   |
 | :-------------------------- | :------------------------------------------------------------------------------- |
 | `campaigns`                 | `content`                                                                        |
-| `companies`                 | `crm.objects.contacts.read`                                                      |
-| `contact_lists`             | `crm.objects.contacts.read`                                                      |
+| `companies`                 | `crm.objects.companies.read`, `crm.schemas.companies.read`                       |
+| `contact_lists`             | `crm.objects.lists.read`                                                         |
 | `contacts`                  | `crm.objects.contacts.read`                                                      |
 | `contacts_list_memberships` | `crm.objects.contacts.read`                                                      |
 | `deal_pipelines`            | either the `crm.objects.contacts.read` scope \(to fetch deals pipelines\) or the `tickets` scope. |
-| `deals`                     | `crm.objects.contacts.read`                                                      |
+| `deals`                     | `crm.objects.deals.read`, `crm.schemas.deals.read`                               |
 | `email_events`              | `content`                                                                        |
-| `engagements`               | `crm.objects.contacts.read`                                                      |
+| `engagements`               | `crm.objects.engagements.read`                                                   |
```

Comment:
For engagements stream `crm.objects.companies.read`, `crm.objects.contacts.read`, `crm.objects.deals.read`, `tickets`, `e-commerce` scopes are reqired - https://legacydocs.hubspot.com/docs/methods/engagements/get-all-engagements